test/everypolitician/popolo/json_test.rb

```diff
@@ -1,16 +1,22 @@
 require 'test_helper'
 
 class Everypolitician::Popolo::JsonTest < Minitest::Test
-  def test_legislative_periods
-    popolo_json = Everypolitician::Popolo::JSON.new(
+  def popolo_json
+    @popolo_json ||= Everypolitician::Popolo::JSON.new(
       events: [
         { name: 'Election 1', classification: 'general election', start_date: '2014-01-01' },
         { name: 'Term 2', classification: 'legislative period', start_date: '2015-01-01' },
         { name: 'Term 1', classification: 'legislative period', start_date: '2010-01-01' },
       ],
     )
+  end
+
+  def test_legislative_periods_ignores_other_event_types
     assert_equal 2, popolo_json.legislative_periods.size
     assert_equal 2, popolo_json.terms.size
```

Comment:
thinking of the tests as a black box (i.e. without knowing the underlying implementation), this assertion (and the parallel one below) come across as a little odd, as you could end up with a _different_ 2-element list for each of these. Perhaps better to test 

```ruby
   assert_equal popolo_json.legislative_periods, popolo_json.terms
```

I'll push a change to do that.